src/remote-docker/remote-docker.yml

```diff
@@ -0,0 +1,97 @@
+# Orb Version 0.1.0
+
+version: 2.1
+description: >
+  Excute Docker build via Artsy-managed Docker daemon with a Circle CI fallback
+
+executors:
+  hokusai:
+    docker:
+      - image: artsy/hokusai:0.5
+
+commands:
+  setup:
+    steps:
+      - add_ssh_keys
+      - checkout
+
+  setup-artsy-remote-docker:
```

Comment:
This command will be namespaced when it's used somewhere else so I don't think it needs to be as verbose.